sdk/core/loader/boot.cc

```diff
@@ -758,6 +758,16 @@ namespace
 	void boot_threads_create(const ImgHdr            &image,
 	                         sched::ThreadLoaderInfo *threadInfo)
 	{
+		Capability<void *> hazardPointers =
+		  build<void *,
+		        Root::Type::RWGlobal,
+		        PermissionSet{Permission::Store,
+		                      Permission::LoadStoreCapability}>(
+		    LA_ABS(__export_mem_hazard_pointers),
+		    LA_ABS(__export_mem_hazard_pointers_end) -
+		      LA_ABS(__export_mem_hazard_pointers));
+		// Space per thread for hazard pointers.
+		static constexpr size_t HazardPointerSpace = 2 * sizeof(void *);
```

Comment:
Could we have a symbolic name for the number of hazard pointers per thread? Could it be configurable?

Reply:
Do you imagine ever supporting more than 2?  I think 2 is both the minimum number to be useful and the maximum number to be performant.

Reply:
Perhaps not, but it would make the code more self-documenting rather than a magic constant. Bonus points if the same name could be used in the linker script.

Reply:
Yup, I agree it should be a symbolic constant, just not sure what scope it needs.  If you agree that we won't change 2, then it can go here.  

Unfortunately, the linker script is generated from lua and we don't have a mechanism for sharing constants between the build system and the code.  It would be nice to add one.

Reply:
Symbolic constant added.